core/services/ocr2/plugins/threshold/decryption_queue.go

```diff
@@ -0,0 +1,236 @@
+package threshold
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/job"
+)
+
+type CiphertextId = []byte
+
+type ThresholdDecryptor interface {
```

Comment:
nit/ this name is repetative when used from outside the package: `threshold.ThresholdDecryptor`. What about `Decryptor`?
```suggestion
type Decryptor interface {
```